src/stores/manager/service.go

```diff
@@ -130,12 +141,17 @@ func (m *BaseManager) GetSecretStore(_ context.Context, name string) (secrets.St
 	return nil, errors.NotFoundError(errMessage)
 }
 
-func (m *BaseManager) GetKeyStore(_ context.Context, name string) (keys.Store, error) {
+func (m *BaseManager) GetKeyStore(ctx context.Context, name string, userInfo *authtypes.UserInfo) (keys.Store, error) {
```

Comment:
I personally think it is acceptable to pass `UserContext` through `context`

If you are completely close to passing it through `context`, I am good to pass it through functions params but we will end up passing it in numerous places.

Reply:
I personally prefer to pass things are arguments than by context in meanwhile it is possible and above all once we interact with our business layer components